src/search/find-visits.js

```diff
@@ -75,42 +77,40 @@ export function findVisitsToPages({pagesResult}) {
      * Raj Pratim Bhattacharya gmail rajpratim1234@gmail.com
      */
     
-    var comp = new Date();
-    var sDate = comp.getTime() - 100*24*60*60*1000 //100 days old search
-    var eDate =  comp.getTime()
-    
+
+    var current_time = new moment().valueOf();
+    var startDate = current_time - 100*24*60*60*1000; //100 days old search
```

Comment:
I meant to say , this commit is made (i.e natural date parser is added ) in the same way I added the date range pickers earlier , as recently @Treora , added some changes to the date-range picker ,in #76 , I will make same changes here to make it consistent wiith the style of implementation.